package exception;

import java.util.Scanner;

public class Test3 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		int num = getNum();
	}
	
	public static int getNum(){
		Scanner input = new Scanner(System.in);//abc
		try{
			int num = input.nextInt();
			return num;
		}catch(Exception e){
			return 0;
		}finally{
			return -1;
		}
	}

}
